templates/default/nginx.conf.erb

```diff
@@ -64,16 +64,6 @@
             # redirect 'group' URLs to organisations
             rewrite ^/group/(aboriginal-and-torres-strait-islander-and-multicultural-affairs|agriculture-fisheries-and-forestry|communities-child-safety-and-disability-services|crime-and-misconduct-commission-queensland|education-training-and-employment|energy-and-water-supply|environment-and-heritage-protection|housing-and-public-works|justice-and-attorney-general|local-government-community-recovery-and-resilience|national-parks-recreation-sport-and-racing|natural-resources-and-mines|police|premier-and-cabinet|public-safety-business-agency|queensland-fire-and-emergency-services|queensland-health|science-information-technology-innovation-and-the-arts|state-development-infrastructure-and-planning|tourism-major-events-small-business-and-the-commonwealth-games|transport-and-main-roads|treasury-and-trade) /organization/$1 permanent;
```

Comment:
should this also be a purl ruling?

Reply:
It's possible. It would potentially be a lot of rules.